.github/workflows/release.yml

```diff
@@ -46,7 +46,6 @@ jobs:
 
   release-container-image:
     needs: release
-    runs-on: ubuntu-latest
```

Comment:
No longer necessary because it's defined in the `release-container-image` workflow?

Reply:
no `runs-on` is just a syntax error when a uses is defined on job level.

Reply:
`runs-on` is only needed if `steps` is defined.